src/api/unified/symbol_manager.cpp

```diff
@@ -168,10 +168,15 @@ AFSymbolManager& AFSymbolManager::getInstance() {
 spdlog::logger* AFSymbolManager::getLogger() { return logger.get(); }
 
 AFSymbolManager::AFSymbolManager()
-    : activeHandle(nullptr)
+    : bkndHandles{}
+    , activeHandle(nullptr)
+    , prevHandle(0)
     , defaultHandle(nullptr)
     , numBackends(0)
+    , newCustomHandleIndex(NUM_BACKENDS)
     , backendsAvailable(0)
+    , activeBackend(AF_BACKEND_CPU)
+    , defaultBackend(AF_BACKEND_CPU)
```

Comment:
No, they've always been there. In fact `af_get_active_backend` and `af::setBackend(AF_BACKEND_DEFAULT)` uses those variables. They're not currently being initialized, so I thought why not initialize them too as a good practice. They're actually being set in the `AFSymbolManager` definition though. I can revert this change though if you think we should leave them not value-initialized.

Reply:
The reason I asked this is, default backend or active backend are automatically chosen if not set by user. So, I am concerned if we will mess that up by setting it to CPU here by default. Can you please verify that.

Reply:
To avoid confusion, lets assign that to `AF_BACKEND_DEFAULT` instead of `CPU`.

Reply:
Yes I can verify that it will choose whatever the `AFSymbolManager` constructor definition determines as the default backend (by the order of CUDA, OpenCL, and CPU) if the user does not explicitly call `af::setBackend`.

I think your suggestion that we initialize it to to `AF_BACKEND_DEFAULT` instead is fine. I see no problems with `activeBackend` and `defaultBackend` being initialized to that in any cases. If there's at least one backend available, both `activeBackend` and `defaultBackend` will be set to that, and if there are no backends available, they will be both set to `AF_BACKEND_DEFAULT`, which I don't think will pose a problem - any arrayfire calls would just fail as expected, unless `af_add/set_backend_library` are called.